README.md

```diff
@@ -55,6 +55,36 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+## Adding Equations
+
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external LaTeX renderer, or insert pre-rendered images in the repo.
+For example, it's possible to insert a link to the online CodeCogs equation editor which will
```

Comment:
I'm not sure we want to recommend the use of external renderers (like CodeCogs). Have you confirmed this with IP/legal?

For example, CodeCogs has a [fair use policy](https://editor.codecogs.com/docs/3-fair_usage.php), but they require we provide a [link back](https://editor.codecogs.com/docs/3-fair_usage.php#link-back).